Makefile

```diff
@@ -1,5 +1,10 @@
 all: test
 
+install:
```

Comment:
cette erreur est apparue suite au make install : 
`openfisca-core 23.1.4 has requirement psutil==5.4.2, but you'll have psutil 5.4.6 which is incompatible`

Reply:
Merci ! L'erreur à été prise en compte ici openfisca/openfisca-core#686

Reply:
Hello!

Désolé, j'arrive après le merge, mais j'essaye de mieux comprendre le contexte de https://github.com/openfisca/openfisca-core/pull/686. 

Je lis dans le `setup.py`: `'psutil >= 5.4.6'` (ce qui revient à demander explicitement une version de `psutil` au supérieure ou égale à `5.4.6`). Qu'est ce qui a poussé à ajouter cette dépendance? Vous avez rencontré des problèmes avec une version antérieure de `psutil`?